src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {
+                new Ball(), new Ball(), new Ball()
+        };
```

Comment:
"create three balls using class Lottery" - but you created them not via Lottery class
```suggestion
```